packages/rn-tester/js/examples/Timer/TimerExample.js

```diff
@@ -86,7 +86,7 @@ class RequestIdleCallbackTester extends React.Component<
         message = 'Burned CPU for 10ms,';
       }
       this.setState({
-        message: `${message} ${deadline.timeRemaining()}ms remaining in frame`,
+        message: `${message} ${deadline.timeRemaining()}ms remaining in frame (timeout: ${deadline.didTimeout})`,
```

Comment:
```suggestion
        message: `${message} ${deadline.timeRemaining()}ms remaining in frame (timeout: ${String(deadline.didTimeout)})`,
```

You need to wrap the boolean to make Flow happy. Also please run prettier on all modified files.